stories/components/STable.01_Playground.story.vue

```diff
@@ -372,6 +372,13 @@ function updateTagsFilter(value: string) {
 .table :deep(.col-tags)      { --table-col-width: 192px; }
 .table :deep(.col-createdAt) { --table-col-width: 192px; }
 
+.table :deep(.col-actions) {
+  --table-col-position: sticky;
+  --table-col-z-index: 50;
+  --table-col-right: 0;
```

Comment:
maybe it is safer to also add `--table-col-left: auto`?

Reply:
ah yeah, wouldn't matter since last column will never reach left (at least till the screen size is not like 100px), but yeah let's add that.

Reply:
Ahhh good point. Yes, this is position: sticky, not absolute. Got confused 😓  But yeah, why not be safe.

Reply:
yeah updated it